src/Context/Drupal/BlockContentContext.php

```diff
@@ -0,0 +1,467 @@
+<?php
+
+namespace Cheppers\DrupalExtension\Context\Drupal;
+
+use Behat\Gherkin\Node\PyStringNode;
+use Behat\Gherkin\Node\TableNode;
+use Behat\Mink\Exception\ExpectationException;
+use Cheppers\DrupalExtension\Context\Base;
+use Drupal\block_content\BlockContentInterface;
+use Drupal\block_content\Entity\BlockContent;
+use Drupal\Core\Entity\EntityInterface;
+use Symfony\Component\Yaml\Yaml;
+use Webmozart\Assert\Assert;
+
+class BlockContentContext extends Base
+{
+    /**
+     * Keep track of block contents so they can be cleaned up.
+     *
+     * @var array
+     */
+    protected $blockContents = [];
+
+    /**
+     * Creates block content of a given type provided in the form:
+     * | info       |
+     * | My info    |
+     * | ...        |
+     *
+     * @Given :type block content:
+     */
+    public function createBlockContents($type, TableNode $blockContentsTable)
+    {
+        foreach ($blockContentsTable->getHash() as $blockContentHash) {
+            $blockContent = (object) $blockContentHash;
+            $blockContent->type = $type;
+            $this->blockContentCreate($blockContent);
+        }
+    }
+
+    /**
+     * Assert viewing block content given its type and info.
+     *
+     * @param string $type
+     *   Block content type machine name.
+     * @param string $info
+     *   Block content info.
+     *
+     * @Given I am visiting the :type block content :info
+     * @Given I visit the :type block content :info
+     */
+    public function iAmViewingTheBlockContent(string $type, string $info)
```

Comment:
Naming convention of methods:
* Method which creates or do something (eg: click on a link) has to start with `do`.
* Method which checks something has to start with `assert`.

See examples in the existing Context classes (./src/Context/Drupal/*)
* \Cheppers\DrupalExtension\Context\Drupal\CoreTabs::assertPrimaryTabsTable()
* \Cheppers\DrupalExtension\Context\Drupal\CoreTabs::doClickPrimaryTab()

Reply:
Fixed in commit 6434a92.